### PR TITLE
3 Line Drawing and 1 Circle Drawing Extra Methods Added

### PR DESCRIPTION
**Pull request by Sukhtaaj**

### CODE REVIEW

File: gui/cadgraphicsscene.h [hunk 1]
```diff
@@ -40,7 +40,8 @@ class CadGraphicsScene : public QGraphicsScene
     explicit CadGraphicsScene(QObject *parent, QUndoStack *);
     enum Mode { NoMode, PointMode, LineMode, CircleMode, EllipseMode, TextMode,
                 ArcMode, ImageMode, DeleteMode, DimHorizontalMode,
-                DimVerticalMode, DimRadialMode, DimDiametricMode };
+                DimVerticalMode, DimRadialMode, DimDiametricMode,
+                oSLine, oMLine, oELine , oCCircle };
```
Comment (kamalpreetgrewal) [inline gui/cadgraphicsscene.h]: Give appropriate meaningful names to modes.

**Comment by kamalpreetgrewal** [comment]
A space should be there before and after operators. This is at a number of places.

File: gui/cadgraphicsscene.h (hunk 1 above)
Comment (bhattigurjot) [inline gui/cadgraphicsscene.h]: See the names of the rest of modes are appropriate and are starting from a capital letter.

**Comment by kamalpreetgrewal** [comment]
Just tested it. Drawing is working appropriately. 
But when the file is saved, the values assigned to ids are incorrect. In some cases, it is garbage value. Only the first id assigned is correct.

**Pull request closed without merging**